### DISCUSSION TITLE
ignore files or directories when adding context via `/file`

### DISCUSSION THREAD

**Discussion by mhuebert** [zed-industries/zed#16631] [Feature Requests]
I'd like to add a directory in my project, but ignore the `node_modules` folder. Is there any way to accomplish this? Perhaps something like `.gitignore` (or an option to respect an existing `.gitignore`). This would also be helpful generally when trying to select files via `/file`; the autocomplete is currently always giving me stuff from `node_modules` which makes it harder to select my own files.

I was able to select a directory containing `node_modules` and then delete it later, but it was very slow (it took some seconds to delete the collapsed folder view).

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
